### DISCUSSION TITLE
How to upgrade from Istio 1.20 (with 1st party jwt) to Istio 1.22 (with 3rd party jwt) with NO impact on the service

### DISCUSSION THREAD

**Discussion by angroszew** [istio/istio#53692] [Q&A]
Is upgrade from Istio 1.20 (with 1st party jwt) to Istio 1.22 (with 3rd party jwt) possible with NO impact on the connections between applications? I did such upgrade but I had to execute kubectl rollout restart after the upgrade to 1.22 for each application to force picking it to the latest proxy.

**Comment by ilrudie** [comment]
For clarity, you are doing in-place upgrades and do not want to use revisioned upgrades. Is that correct?

**Reply by angroszew** [reply]
yes, correct
